playbooks/oawaiver.yml

```
@@ -0,0 +1,24 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
+# to run in production, pass '-e runtime_env=production'
+- name: Build oawaiver infrastructure
+  hosts: oawaiver_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/oawaiver/{{ runtime_env | default('staging') }}.yml
+    - ../group_vars/oawaiver/vault.yml
+  roles:
+    - role: roles/oawaiver
+      when: runtime_env | default('staging') == "production"
```

Probably this relates to the Datadog role.
```suggestion
   - role: roles/datadog
      when: runtime_env | default('staging') == "production"
```